components/editor/ai-integration/integration-with-aiprompt.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Integration with AIPrompt
+page_title: Editor Integration with AIPrompt
+description: Integration of the AIPrompt component in the Editor for Blazor.
+slug: editor-aiprompt-integration
+tags: telerik,blazor,editor,ai,ai integration,aiprompt
+published: True
+position: 3
+---
+
+# Editor Integration with AIPrompt
+
+The Editor provides a built-in integration with the [AIPrompt component](slug:aiprompt-overview) to help users add AI-driven suggestions, completions, and assistance to their Editor.
+
+The integration with the AIPrompt component covers the following use cases:
+
+* Standalone prompting - The user can ask the AIPrompt without any context provided by the Editor. This could be useful for ideas generation, writing a whole piece of content (blog post), or just random questions.
```

Comment:
Nouns used as adjectives must be singular.

```suggestion
* Standalone prompting - The user can ask the AIPrompt without any context provided by the Editor. This could be useful for idea generation, writing a whole piece of content (blog post), or just random questions.
```